timescaledb/getting-started/query-data.md

```diff
@@ -1,153 +1,145 @@
-powerful function for time-series analysis is `time_bucket_gapfill`.
-
-### time_bucket_gapfill()
-
-Another common problem in time-series analysis is dealing with imperfect datasets.
-Some time-series analyses or visualizations want to display records for each
-selected time period, even if no data was recorded during that period. This is
-commonly termed "gap filling," and may involve performing such operations as
-recording a "0" for any missing data, interpolating missing values, or carrying
-the last observed value forward until new data is recorded.
-
-Timescale provides [`time_bucket_gapfill()`],
-[`locf()`], and [`interpolate()`] to help perform analysis on data with gaps.
-
-In the sample dataset, there are days with no rain or snow for some cities.
-However, you might still want to perform an analysis or graph a trend line about
-rain or snow for a particular time period.
-
-For example, here's a query which calculates the total snowfall for each city in
-30-day time periods for the past year:
-
-```sql
--- non-gapfill query
-SELECT time_bucket('30 days', time) as bucket,
-   city_name, sum(snow_1h_mm) as sum
-   FROM weather_metrics
-   WHERE time > now() - INTERVAL '1 year' AND time < now()
-   GROUP BY bucket, city_name
-   ORDER BY bucket DESC;
-```
-
-Notice that the results only include time periods for when cities have snowfall,
-rather than the specific time period of our analysis, which is one year.
-
-To generate data for all the time buckets in our analysis period, we can use
-time_bucket_gapfill instead:
-
-```sql
------------------------------------------
--- time_bucket_gapfill
--- total snow fall per city
--- in 30-day buckets for past 1 year
------------------------------------------
-SELECT time_bucket_gapfill('30 days', time) as bucket,
-   city_name, sum(snow_1h_mm) as sum
-   FROM weather_metrics
-   WHERE time > now() - INTERVAL '1 year' AND time < now()
-   GROUP BY bucket, city_name
-   ORDER BY bucket DESC;
-```
-
-TimescaleDB SQL functions like time_bucket and time_bucket_gapfill are helpful
-for historical analysis of your data and creating visuals with specific time periods.
-
-Now that you're equipped with the basics of `time_bucket`, let's learn about
-continuous aggregates in the next section.
-
-
-
-[`time_bucket()`]: /api/:currentVersion:/hyperfunctions/time_bucket
-[`time_bucket_gapfill()`]: /api/:currentVersion:/hyperfunctions/gapfilling-interpolation/time_bucket_gapfill
-[`last()`]: /api/:currentVersion:/hyperfunctions/last
-[`first()`]: /api/:currentVersion:/hyperfunctions/first
-[`locf()`]: /api/:currentVersion:/hyperfunctions/gapfilling-interpolation/locf
-[`interpolate()`]: /api/:currentVersion:/hyperfunctions/gapfilling-interpolation/interpolate
+analysis in fewer lines of code. Let's look at how to use three of these functions. 
+
+Functions covered include:
+ * [first()][first]: used for finding the earliest value based on a time within an aggregate group
```

Comment:
I'm totally good with whatever the standard is for the docs. It's helps me to see brackets because it indicates that it is a function, but I also am good with whatever!